### ISSUE TITLE
.exe dose not launch

### ISSUE THREAD

**Issue by TheRodrixHD** [BetterDiscord/Installer#46]
**Describe the bug**
When I try to launchBetterDiscord.exe does not launch 

https://user-images.githubusercontent.com/68759622/113475135-6bc72180-947c-11eb-8a7d-34d8d624556c.mp4

**Comment by Ynaught** [comment]
I am having the same issue. Any specs or something I can offer to help better diagnose the issue?

**Comment by rauenzi** [comment]
Check that it was not blocked by whatever antivirus you have installed

**Comment by rauenzi** [comment]
Duplicate of #43
